ProjectStructureAndRoles.md

```diff
@@ -75,17 +70,17 @@ A Project participant may resign by notifying the team mailing list and by sendi
 
 **Working Group participants**
 
-Each Project participant can be part of one or more of the Working Groups of the Project by adding his name to the WG_PARTICIPANTS.MD files. By that he gets the role Working Group participant.
+Each Project participant can be part of one or more of the Working Groups of the Project by subscribing to the mailing list(s). By that he gets the role Working Group participant, the WG_PARTICIPANTS.MD file(s) is/are then also adjusted by the administrators of the Project.
 
-A Working Group participant may resign by removing his name of the WG_PARTICIPANTS.MD file.
+A Working Group participant may resign by unsubscribing from the mailing list(s), the WG_PARTICIPANTS.MD file(s) is/are adjusted by the administrators of the Project.
 
 **Contributors**
 
-Each Project participant can contribute to the Sub Projects of the Project. Contributors to a Sub Project can contribute by creating pull requests. There’s no formal process for becoming or resigning as Contributor. The list of Contributors of a specific Sub Project is maintained by GitHub.
+Each Project participant can contribute to the Sub Projects of the Project. Contributors to a Sub Project can contribute by creating pull requests. To become or resign as Contributor please subscribe / unsubscribe to the mailing list of the Sub Project. The list of Contributors of a specific Sub Project is maintained by the administrators of the Project using GitHub functionality.
```

Comment:
It is said that "The list of Contributors of a specific Sub Project is maintained by the administrators of the Project using GitHub functionality". Why contributors are maintained by CAMARA administrators rather than Sub Project maintainers?

Reply:
That is due to the administration model of GitHub. Only admins of the whole Project !!! can add or remove people